react/modules/module01/configs.md

```diff
@@ -1,6 +1,6 @@
-# Конфигурация eslint и prettier для использования в задании
+# Eslint and prettier configurations
 
-## Набор рекомендуемых библиотек
+## Recommended libraries (versionS can be different)
 
 ```
     "@typescript-eslint/eslint-plugin": "^5.22.0",
```

Comment:
Since we have ```husky``` and ```lint-staged``` dependencies, shouldn't we provide an example of ```package.json``` that uses them to run ```eslint``` and ```prettier``` when we try to commit changes

Reply:
We should probably add ```eslint-plugin-react-hooks``` into these list, because when I use vite, I see this message
![image](https://user-images.githubusercontent.com/69504277/224490310-bc03559b-7357-47df-8e1d-beeb386aa45c.png)

But CRA comes with this dependency and it's not recommended to install it separately
https://www.npmjs.com/package/eslint-plugin-react-hooks
![image](https://user-images.githubusercontent.com/69504277/224490427-341da17c-a305-466c-8d29-3e1be9262e8b.png)

Reply:
I guess we could provide a link to the article which explains how one can setup husky, this one looks good https://www.freecodecamp.org/news/how-to-add-commit-hooks-to-git-with-husky-to-automate-code-tasks/

Reply:
Modified suggestions, please check. Also I think we should allow minor changes in eslint config:
```
Might be insignificantly changed
```